src/Microsoft.OpenApi.CSharpAnnotations.DocumentGeneration/OpenApiGeneratorConfig.cs

```diff
@@ -67,6 +73,7 @@ public class OpenApiGeneratorConfig
             }
 
             OpenApiDocumentVersion = openApiDocumentVersion;
+            SchemaGenerationSettings = schemaGenerationSettings;
```

Comment:
>schemaGenerationSettings [](start = 39, length = 24)

Check for null and use the default if so?

Reply:
Will just check for null, b/c default could be used with above constructor.

---
In reply to: [195170401](https://github.com/Microsoft/OpenAPI.NET.CSharpAnnotations/pull/133#discussion_r195170401) [](ancestors = 195170401)